fala/assets-src/sass/_fala-results.scss

```diff
@@ -1,19 +1,34 @@
 // Map styling
 .gm-style {
+  iframe ~ div div:focus-within {
+      background-color:$govuk-focus-colour;
+      box-shadow:0 -2px 0 0 $govuk-text-colour inset;
+  }
+  iframe ~ div div a:focus {
+      outline:none!important;
+      background-color:transparent!important;
```

Comment:
Not sure if I wasted my time tbh as if these are the GDS design system changes, we don't really want to add/remove anything.

But I just wanted to highlight that `on focus` of the `a` tag, the background-color does not become transparent, but rather stays yellow because of the class selector above.

Example:

![Screenshot 2020-11-19 at 11 45 08 (2)](https://user-images.githubusercontent.com/64010092/99662122-bd868680-2a5c-11eb-89a4-3d103c16a438.png)

Or is this normal and I just wasted my time :(